src/test/java/uk/gov/companieshouse/overseasentitiesapi/mocks/RemoveMock.java

```diff
@@ -1,9 +0,0 @@
-package uk.gov.companieshouse.overseasentitiesapi.mocks;
-
-import uk.gov.companieshouse.overseasentitiesapi.model.dto.RemoveDto;
-
-public class RemoveMock {
-    public static RemoveDto getRemoveDto() {
```

Comment:
Called from one place. Less complexity and lines of code to just instantiate this object when it's needed.